src/realm/error_codes.h

```diff
@@ -266,9 +266,16 @@ typedef enum realm_sync_errno_session {
     RLM_SYNC_ERR_SESSION_REVERT_TO_PBS = 234,
     RLM_SYNC_ERR_SESSION_BAD_SCHEMA_VERSION = 235,
     RLM_SYNC_ERR_SESSION_SCHEMA_VERSION_CHANGED = 236,
+    RLM_SYNC_ERR_SESSION_FIELD_LVL_PERMS_NOT_SUPPORTED = 238,
+    RLM_SYNC_ERR_SESSION_SCHEMA_VERSION_FORCE_UPGRADE = 239,
     // Error code 299 is reserved as an "unknown session error" in tests
 } realm_sync_errno_session_e;
 
+// These errors are intended for the edge server and are an error if received by a sync client
+typedef enum realm_sync_errno_edge_e {
```

Comment:
I don't think we should expose errors for the edge server. The only way we could get these is via a bug. I think getting an UnkownError for edge server only errors would be totally acceptable.